sections/intro.tex

```diff
@@ -1,6 +1,290 @@
 \section{Introduction}\label{sec:intro}
 
-This introduction will briefly summarise the literature review for studying a
-patient corpus via clustering. Following this, a condensed data analysis is
-presented highlighting the main conclusions of the clustering and the overall
-benefits compared with traditional condition-treatment segmentation.
+Population health research is becoming increasingly based on data-driven methods
+(as opposed to those designed solely by clinical experts) for patient-centred
+care through the advent of accessible software and a relative abundance of
+electronic data. A vital part of such research is to better understand the
+healthcare needs and behaviours of a population, and it can be beneficial to
+find an appropriate segmentation of that population; such a segmentation allows
+for finer-grained analysis of groups in the population that share some form of
+homogeneity. One commonly used method for such patient-centred analysis is that
+of patient flow and their interaction with the healthcare system.
+
+However, this process relies heavily on detailed data --- about both the system
+and the population within that system --- which may limit research where
+sophisticated data pipelines are not yet in place. This work demonstrates how
+this issue may be overcome using administrative, spell-level hospital data to
+build a patient clustering that feeds into a multi-class queuing model.
+Specifically, this work examines patient records from the National Health
+Service (NHS) Wales Cwm Taf Morgannwg University Health Board (UHB) that present
+chronic obstructive pulmonary disease (COPD). COPD is of particular interest to
+Cwm Taf Morgannwg UHB as the condition is known to often present as a
+comorbidity in patients~\cite{Houben2019} and it was found that they had the
+highest prevalence of the condition across all the Welsh health boards in an
+internal report by NHS Wales.
+
+% TODO No reference for graph sent by Kendal but is this alright?
```

Comment:
Can you remind me what this is about?

Sounds like we'll need to at least check with Kendal but at best reference some original source.

Reply:
It was to do with a figure that didn't come from a published article per se -- just an NHS report. I need to get in touch with Kendal anyway so I'll ask for his recommendation.

Reply:
I'd wait on this, it'll come out in the wash when we have a draft for Kendal to look at.

Reply:
Right you are